data/SinghRamesh_2023/metadata.yml

```diff
@@ -0,0 +1,459 @@
+source:
+  primary:
+    key: SinghRamesh_2023
+    bibtype: Article
+    year: '2023'
+    author: Arun Singh Ramesh and Alexander W. Cheesman and Habacuc Flores-Moreno
+      and Noel D. Preece and Darren M. Crayn and Lucas A. Cernusak
+    journal: Frontiers in Forests and Global Change
+    title: Temperature, nutrient availability, and species traits interact to shape
+      elevation responses of Australian tropical trees
+    volume: '6'
+    doi: 10.3389/ffgc.2023.1089167
+contributors:
+  data_collectors:
+  - last_name: Singh Ramesh
+    given_name: Arun
+    ORCID: 0000-0002-9143-5514
+    affiliation: College of Science and Engineering, James Cook University, Cairns,
+      QLD, Australia
+    additional_role: contact
+  - last_name: Cernusak
+    given_name: Lucas
+    ORCID: 0000-0002-7575-5526
+    affiliation: College of Science and Engineering, James Cook University, Cairns,
+      QLD, Australia
+  austraits_curators: Elizabeth Wenk
+dataset:
+  data_is_long_format: no
+  custom_R_code: '
+    data %>%
+      mutate(
+        location = "Environmental Research Complex, JCU, Cairns",
+        leaf_N_per_area = `%N`*LMA/100
+      )
+  '
+  collection_date: unknown/unknown
+  taxon_name: Species_ID
+  location_name: location
+  individual_id: POT
+  description: A glasshouse experiment investigating the effects of temperature, soil
+    nutrient availability, and species on plant functional traits.
+  basis_of_record: experiment
+  life_stage: sapling
+  sampling_strategy: We collected seeds from the four focal species, with each species
+    collected at a single site (Supplementary Table 1). The four collection sites
+    were distributed across most of the elevation range found in the Australian Wet
+    Tropics (Lowland species, F. ifflana ~300 m and F. bourjotiana ~600 m, and Upland
+    species F. brayleyana ~990 m and F. oppositifolia ~1,550 m). The seeds were allowed
+    to germinate under ambient shade-house conditions at the Environmental Research
+    Complex (ERC), (~30 m a.s.l.) JCU, Cairns, Australia. Healthy saplings were transplanted
+    (approximately 3 months after sowing seeds) into either 13.5 L pots (Garden City
+    Pots, Model P300ST00) containing nutrient-rich soil (NR), comprising a 1:1 mixture
+    by volume of premium garden potting mix and compost (Northside Landscape Supplies
+    Pty. Ltd., Trinity Beach); or 8.5 L pots (Garden City Pots, Model P250STTL) containing
+    nutrient-poor soil (NP). The nutrientpoor soil comprised a locally collected dermasol,
+    low in organic matter and supplemented with perlite and washed river sand to improve
+    drainage (Supplementary Table 2). The two soil mixtures, nutrient-rich and nutrient-poor,
+    had similar d15N at the start of the experiment, based on measurements of subsamples;
+    these values were 5.4 and 5.3h, respectively.  ^p  The experimental pots were
```

Comment:
I don't know why but the parts-per-thousand symbol gets copied and pasted as 'h' in "5.3h".

Reply:
Fixed